src/VisualStudio/CSharp/Impl/VSPackage.resx

```diff
@@ -363,4 +363,28 @@ Show items from unimported namespaces (experimental);</value>
   <data name="An_empty_CSharp_script_file" xml:space="preserve">
     <value>An empty C# script file.</value>
   </data>
+  <data name="Only_take_effect_when_show_completion_list_after_a_character_is_typed_is_enabled" xml:space="preserve">
```

Comment:
These resources are needed. Right now Unified Settings support load resources from package resources for EnumLabel.
(They have a bug prevent us from loading from any dll, https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/524045)